packages/amplify-category-notifications/src/migrations/index.ts

```diff
@@ -5,7 +5,7 @@ import { invokeAnalyticsMigrations } from '../plugin-client-api-analytics';
  * checks if the project has been migrated to the latest version
  */
 export const migrationCheck = async (context: $TSContext): Promise<void> => {
-  if (['add', 'update', 'push'].includes(context.input.command)) {
+  if (['add', 'configure', 'push'].includes(context.input.command)) {
```

Comment:
would it make sense to keep update in case of alias evolves to something different?

Reply:
updated